SwissTransferCoreUI/Components/Thumbnail/LargeThumbnailView.swift

```diff
@@ -22,31 +22,76 @@ import SwiftUI
 import SwissTransferCore
 
 public struct LargeThumbnailView: View {
```

Comment:
Can we use another name? Thumbnail refers only to the preview of the image, not the full component